src/test/java/com/epam/rp/api/tests/ApacheApiTest.java

```diff
@@ -0,0 +1,275 @@
+package com.epam.rp.api.tests;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.apache.http.client.methods.HttpPut;
+import org.apache.http.entity.StringEntity;
+import org.apache.http.impl.client.HttpClients;
+import org.apache.http.util.EntityUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.UUID;
+
+public class ApacheApiTest extends ApiBaseTest {
+
+    @Test
+    public void testCreateDashboard_ValidParameters() throws IOException {
+        HttpClient client = HttpClients.createDefault();
+        HttpPost postRequest = new HttpPost(baseUrl + "/api/v1/" + projectName + "/dashboard");
+        postRequest.setHeader("Authorization", "Bearer " + apiToken);
+        postRequest.setHeader("Content-type", "application/json");
+        String randomDescription = "Desc " + UUID.randomUUID();
+        String randomName = "Name " + UUID.randomUUID();
+        String jsonBody = String.format("{ \"description\": \"%s\", \"name\": \"%s\" }", randomDescription, randomName);
+        postRequest.setEntity(new StringEntity(jsonBody));
+        HttpResponse response = client.execute(postRequest);
+        int statusCode = response.getStatusLine().getStatusCode();
+
+        Assert.assertEquals(statusCode, 201);
+    }
+
+    @Test
+    public void testCreateDashboard_EmptyDashboardName() throws IOException {
+        HttpClient client = HttpClients.createDefault();
+        HttpPost postRequest = new HttpPost(baseUrl + "/api/v1/" + projectName + "/dashboard");
+        postRequest.setHeader("Authorization", "Bearer " + apiToken);
+        postRequest.setHeader("Content-type", "application/json");
+        String jsonBody = String.format("{ \"description\": \"Test description\", \"name\": \"\" }");
+        postRequest.setEntity(new StringEntity(jsonBody));
+        HttpResponse response = client.execute(postRequest);
+        int statusCode = response.getStatusLine().getStatusCode();
+
+        Assert.assertEquals(statusCode, 400);
+        String responseBody = EntityUtils.toString(response.getEntity());
+        ObjectMapper objectMapper = new ObjectMapper();
+        JsonNode rootNode = objectMapper.readTree(responseBody);
+        String errMessage = rootNode.path("message").asText();
```

Comment:
This lines can be moved to separate method and reused across the tests.

```java
private String getBodyNode(HttpsResponse response, String path) {
        String responseBody = EntityUtils.toString(response.getEntity());
        ObjectMapper objectMapper = new ObjectMapper();
        JsonNode rootNode = objectMapper.readTree(responseBody);
        return rootNode.path(path).asText();
}
```

Reply:
Hi Pavel, it's updated, thank you